src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -48,14 +49,18 @@ function onClick(store: Store) {
         percentAdequatePopulation,
         percentInadequatePopulation
       } = summaryStatistics([_], store)
+      let specialty = store.get('providers')[0].specialty // TODO: Is this safe to assume?
+      if (specialty == null) {
+        specialty = 'Undefined'
+      }
       return [
         representativePoint.county,
         representativePoint.zip,
-        store.get('providers')[0].specialty, // TODO: Is this safe to assume?
-        numAdequatePopulation,
-        numInadequatePopulation,
-        percentAdequatePopulation,
-        percentInadequatePopulation,
+        specialty,
+        formatNumber(numAdequatePopulation),
+        formatNumber(numInadequatePopulation),
+        formatPercentage(percentAdequatePopulation),
+        formatPercentage(percentInadequatePopulation),
```

Comment:
I don't think we want to format these in the csv. It will make it harder to analyze in excel.

Reply:
Done, good point